scripts/pre_commit_linter.py

```diff
@@ -1159,6 +1159,8 @@ def _check_docstrings(all_files):
     previous_line_message = (
         'There should not be any empty lines before the end of '
         'the multi-line docstring.')
+    space_in_docstring_message = (
```

Comment:
Change this to `space_after_triple_quotes_in_docstring_message`

Reply:
Done